vendor/github.com/heketi/heketi/client/api/go-client/client.go

```diff
@@ -25,7 +25,7 @@ import (
 	"strconv"
 	"time"
 
-	jwt "github.com/dgrijalva/jwt-go"
```

Comment:
why aren't we seeing this dropped from anywhere?

Reply:
If I understand your question correctly, you are asking if this is a transitive dependency for heketi why is it not dropped in go.mod in PR. The reason is, it is a transitive dependency for two testing package(s) so it still lingers in auto-generated go.mod of k8s

```
x@xkubernetes % go mod graph | grep github.com/dgrijalva/jwt-go
go.etcd.io/etcd@v0.5.0-alpha.5.0.20200910180754-dd1b699fc489 github.com/dgrijalva/jwt-go@v3.2.0+incompatible
github.com/spf13/viper@v1.7.0 github.com/dgrijalva/jwt-go@v3.2.0+incompatible

x@x kubernetes % go mod why github.com/dgrijalva/jwt-go
# github.com/dgrijalva/jwt-go
k8s.io/kubernetes/pkg/registry/registrytest
k8s.io/apiserver/pkg/storage/etcd3/testing
go.etcd.io/etcd/etcdserver
go.etcd.io/etcd/auth
github.com/dgrijalva/jwt-go

x@x kubernetes % go mod why github.com/spf13/viper       
# github.com/spf13/viper
k8s.io/kubernetes/test/e2e
github.com/spf13/viper
```